doc/api/crypto.md

```diff
@@ -172,14 +172,14 @@ console.log(encrypted);
 // Prints: ca981be48e90867604588e75d04feabb63cc007a8f8ad89b10616ed84d815504
 ```
 
-### cipher.final([output_encoding])
+### cipher.final([outputEncoding])
```

Comment:
I didn't realise that when I started, now I know I think you're quite right. It would be nice if everything was camelCase, but if everyone is happy for there to be this inconsistency then I'll revert this back.